src/Uno.UI/Controls/NavigationBarHelper.Android.cs

```diff
@@ -0,0 +1,71 @@
+﻿#if __ANDROID__
+using System;
+using Android.App;
+using Android.Graphics;
+using Android.Util;
+using Android.Views;
+using Window = Windows.UI.Xaml.Window;
+
+namespace Uno.UI.Controls
+{
+	public static class NavigationBarHelper
```

Comment:
Is this supposed to be useable outside of Uno ? Otherwise, make it internal.

Reply:
No, I've made it internal